solidity/test/ConverterLiquidity.js

```diff
@@ -318,17 +316,14 @@ describe('ConverterLiquidity', () => {
         );
     };
 
-    const getLiquidityReturns = async (firstTime, converter, reserveTokens, reserveAmounts) => {
+    const getLiquidityReturn = async (firstTime, converter, reserveTokens, reserveAmounts) => {
         if (firstTime) {
             const length = Math.round(
                 reserveAmounts.map((reserveAmount) => reserveAmount.toString()).join('').length / reserveAmounts.length
             );
-            const retVal = new BN('1'.padEnd(length, '0'));
-            return reserveAmounts.map((reserveAmount, i) => retVal);
+            return new BN('1'.padEnd(length, '0'));
```

Comment:
Can you use `pow` instead of string padding?

Reply:
done